Formula/llvm.rb

```diff
@@ -108,11 +62,6 @@ class Llvm < Formula
   uses_from_macos "ncurses"
   uses_from_macos "zlib"
 
-  patch :p1 do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/c7da61787c62822804dd90192dd9dd254511192c/llvm/10.0.1-llvm.diff"
-    sha256 "86b4e2bf10dfb9a04153e3d4fc744c3f323f3e13fc7b3a7d22e6791d47eb2e2c"
-  end
```

Comment:
A companion pull request (after this is merged) to homebrew/formula-patches to delete this would be appreciated!

Reply:
PR created: Homebrew/formula-patches#311